tests/component/test_stream_readers_ai.py

```diff
@@ -110,6 +136,25 @@ def test___analog_single_channel_reader___read_many_sample_with_wrong_dtype___ra
     assert "float64" in exc_info.value.args[0]
 
 
+@pytest.mark.grpc_skip(reason="internal_read_analog_waveform_ex not implemented in GRPC")
```

Comment:
I think this function was added in NI-DAQmx 24.5 or 24.8. 
- Should the tests be marked accordingly or do we always run a newer version than this?
- I guess LibImporter already handles the error case.